client/lib/plugins/utils.js

```diff
@@ -146,7 +146,9 @@ export function normalizeCompatibilityList( compatibilityList ) {
 export function normalizePluginData( plugin, pluginData ) {
 	plugin = getAllowedPluginData( assign( plugin, pluginData ) );
 
-	return transform( plugin, function ( returnData, item, key ) {
+	return Object.keys( plugin ).reduce( ( returnData, key ) => {
+		const item = plugin[ key ];
```

Comment:
You can also do `Object.entries( plugin ).reduce( ( returnData, [ key, item ] ) => { ... } )`

Reply:
Good call. Done in b60492fc2a10049df29f005cf9b02163789fd017 for both instances.